index.html

```diff
@@ -785,6 +785,23 @@
           "site preferences".
         </p>
       </section>
+      <section>
+        <h3>
+          Managing source swapping between breakpoints
+        </h3>
+        <p>
+          When a new image is loaded via &lt;source&gt; after hitting a new
+          breakpoint, there is a delay while the image is loaded. Any CSS which
+          matches the breakpoint applies itself. However, we're stuck with the
+          previous image while the new one loads. If we had a JavaScript event
+          fire when the new image is loaded, loadstart for example. The user
```

Comment:
You want to avoid talking about a solution here, just the problem you're trying to solve.